src/tools/components/APIRequest.tsx

```diff
@@ -67,29 +125,37 @@ const APIRequest: FC<APIRequestProps> = ({ inputParamsData, jobType }) => {
         , you could run a new job on the command line with the same input as
         this job by running:
       </p>
-      {loading ? (
+      {inputParamsData &&
+      'loading' in inputParamsData &&
+      inputParamsData.loading ? (
         <Loader />
       ) : (
-        <CodeBlock lightMode>{inputToCurl(data, jobType)}</CodeBlock>
+        <CodeBlock lightMode>{inputToCurl(jobType, data)}</CodeBlock>
       )}
-      {'sequence' in data && data.sequence.includes("'") && (
+      {jobType === JobTypes.PEPTIDE_SEARCH && (
+        <p>
+          <small>
+            The URL with your job result will be listed in the{' '}
+            <code>Location</code> response header
+          </small>
+        </p>
+      )}
+      {data && 'sequence' in data && data.sequence.includes("'") && (
         <p>
           <small>
             You might need to escape the sequence as it contains special
             characters
           </small>
         </p>
       )}
-      <p>
-        You can refer to the documentation for these values on the{' '}
-        <a
-          target="_blank"
-          rel="noreferrer noopener"
-          href={documentation.get(jobType)}
-        >
-          API documentation page
-        </a>
-      </p>
+      {documentationURL && (
+        <p>
```

Comment:
<img width="2160" alt="Screenshot 2022-03-14 at 08 47 11" src="https://user-images.githubusercontent.com/4357962/158136719-340ec0b5-181c-4eff-830f-b6ef70e11b0f.png">
Could be a little easier to spot the API link sentence with a some white space between it and the code block above.